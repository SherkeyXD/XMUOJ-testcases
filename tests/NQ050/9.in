6
16 802 136 491 740 815 16 352 672 50 886 765 510 570 980 508 594 
28 234 411 47 360 920 430 65 309 84 950 792 212 918 869 102 170 368 388 554 456 20 392 374 365 480 844 979 813 
25 208 110 763 170 400 537 863 968 437 56 408 97 498 344 280 387 906 629 725 529 635 880 358 48 985 
13 192 128 140 361 716 313 350 254 934 336 477 824 957 
25 376 968 712 99 180 74 710 214 480 857 192 764 527 104 32 196 968 152 16 302 250 645 48 838 427 
20 600 322 825 138 922 510 512 772 398 543 405 155 544 760 147 158 950 680 698 688 
