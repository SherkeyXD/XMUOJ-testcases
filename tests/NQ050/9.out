16 50 136 352 491 508 510 570 594 672 740 765 802 815 886 980
20 47 65 84 102 170 212 234 309 360 365 368 374 388 392 411 430 456 480 554 792 813 844 869 918 920 950 979
48 56 97 110 170 208 280 344 358 387 400 408 437 498 529 537 629 635 725 763 863 880 906 968 985
128 140 192 254 313 336 350 361 477 716 824 934 957
16 32 48 74 99 104 152 180 192 196 214 250 302 376 427 480 527 645 710 712 764 838 857 968 968
138 147 155 158 322 398 405 510 512 543 544 600 680 688 698 760 772 825 922 950
